config/krakend/templates/inventory.tmpl

```diff
@@ -0,0 +1,59 @@
+{{range $index, $element := .}}
+{{if gt $index 0}},{{end}}
+{
+    "endpoint": "{{ .endpoint }}",
+    "method": "{{ .method }}",
+    "querystring_params": ["*"],
+    "output_encoding": "no-op",
+    "headers_to_pass": ["*"],
+    "backend": [
+    {
+        "url_pattern": "{{ .url_pattern }}",
+        "encoding": "no-op",
+        "sd": "static",
+        "disable_host_sanitize": false,
+        "extra_config": {
+            "github.com/devopsfaith/krakend-martian": {
+                "fifo.Group": {
+                    "scope": ["request", "response"],
+                    "aggregateErrors": true,
+                    "modifiers": [
+                        {
+                            "header.Modifier": {
+                                "scope": ["request"],
+                                "name": "Content-Type",
+                                "value":  "application/json"
+                            }
+                        },
+                        {
+                            "header.Modifier": {
+                                "scope": ["request"],
+                                "name": "x-tenant-id",
+                                "value":  "frinx"
```

Comment:
the default multitenancy values are also used when accessing workflow-proxy and resource-manager right ? 

so we probably have them copy-pasted in a few places already, can we extract them into global env vars ?

Reply:
KrakenD can't use env variables in nested configs. But common headers can be stored in 'partials' and then inserted into specific templates.